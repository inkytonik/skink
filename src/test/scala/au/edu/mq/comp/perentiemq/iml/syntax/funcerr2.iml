// No return variable
def funcerr2(x): () {
}

