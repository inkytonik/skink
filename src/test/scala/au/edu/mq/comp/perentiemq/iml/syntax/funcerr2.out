src/test/scala/au/edu/mq/comp/perentiemq/iml/syntax/funcerr2.iml:2:19: identifier expected
def funcerr2(x): () {
                  ^
